### PR TITLE
Add bug report, feature request and pull request templates for GitHub

### LINKED ISSUE #13: Add issue and PR templates
Look at our other projects and decide which parts to adopt for this project

### PR DESCRIPTION
**Pull request by JoeyStk**
This PR adds GitHub templates for bug reports, feature requests and pull requests.

- Create a .github folder
- Create three md files with the template

The templates are the same as we use in Lunes CMS and Integreat CMS. I like them as they are, that's why I didn't change them. However since we're leaning stronger towards TDD it might be interesting to hear what you think about it

Fixes #13

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+### Alternatives
+<!-- A clear and concise description of any alternative solutions or features you've considered, and why you're proposed solution is better. -->
+
+
+### Additional Context
+<!-- Add any other information or screenshots about the feature request here. -->
+
+
+### Design Requirements
+<!-- If the customization includes input from our design team, the detailed requirements will be collected here. Note: These will exist mainly in German to simplify internal communication. -->
```
Comment (charludo) [inline .github/ISSUE_TEMPLATE/feature-request.md:30]: Are we going to be working with the design team on this? (Genuine question, I do not know)

Comment (JoeyStk) [reply .github/ISSUE_TEMPLATE/feature-request.md:30]: I think we'll be using the existing mockups that we already have from the pre-existing Figma file. I was planning to add screenshots for every issue that has something to do with templates, but if everyone has access to the link, I guess you're right and we can skip this part

**Review by charludo — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (timoludwig) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Hmm, the "Additional Context" section already mentiones screenshots. I don't think we need a separate section for this..

Comment (JoeyStk) [reply .github/ISSUE_TEMPLATE/feature-request.md]: That is a valid point. However I like that the new section is more prominent (and hopefully reminds people to also upload a screenshot more often) than just Additional Context. Can I convince you to remove the "or screenshots" from Additional Context and keep it as two sections. Additional Context then being more for written, informational background context and Screenshots solely for screenshots. If not, I'm also fine with removing the section

Comment (timoludwig) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Sure, would be fine for me.

**Review by timoludwig — APPROVED** [review]

**Pull request merged**